src/components/facet-browser/FacetBrowserModalBody.tsx

```diff
@@ -30,6 +31,25 @@ const FacetBrowserModalBody: React.FunctionComponent<
       setOpenFacets([...openFacets, facet]);
     }
   };
+  const { track } = useStatistics();
+
+  useEffect(() => {
```

Comment:
Consider if this should use `useDeepCompareEffect`.

This seems like a `useEffect` which depends on a non-primitive data structure which may point to different values but having equal values.

Reply:
I went through the tracking around the apps and we now officially use useDeepCompareEffect 😉 
Had to merge develop branch in first though.